AWSIoT/Internal/AWSIoTMQTTClient.h

```diff
@@ -90,6 +90,8 @@
 
 @property(atomic, assign) BOOL isMetricsEnabled;
 @property(atomic, assign) NSUInteger publishRetryThrottle;
+@property(atomic, assign) NSString *SDK_VERSION;
```

Comment:
Lets not expose the SDK_VERSION as a property in the .h file. It is hard coded in the .m file and is not intended to be set.  Having this be both a property and internal constant is confusing.

Instead, just move the hard coded constant to the AWSIoTDataManager.m file as that is the only place where it is being used.

Reply:
Done